pydoctor/astbuilder.py

```diff
@@ -518,7 +518,7 @@ def _handleModuleVar(self,
         parent = self.builder.current
         obj = parent.resolveName(target)
         
-        if obj is None:
+        if obj is None or isinstance(obj, model.Module):
```

Comment:
```suggestion
        if obj is None or obj.fullName() != f'{parent.fullName()}.{target}':
```

Maybe this check is more generic, what do you think @adiroiban ?

Reply:
I find that check harder to understand, so if we were to use it I think it would need to be explained in a comment.

Reply:
Ok, now that I think about it, this is probably not the best to handle this situation. 
I'm not sure exactly why `resolveName` is used in this context. I think we should avoid using `resolveName` from the builder since it expect all object in the tree to be already created in order to work correctly. 
I've edited this suggestion.